src/widget/wsearchlineedit.cpp

```diff
@@ -97,6 +97,7 @@ WSearchLineEdit::WSearchLineEdit(QWidget* pParent)
 
     m_clearButton->setCursor(Qt::ArrowCursor);
     m_clearButton->setObjectName(QStringLiteral("SearchClearButton"));
+    m_clearButton->setFocusPolicy(Qt::ClickFocus);
```

Comment:
I don't agree:
how do you _quickly_ clear the search from a controller now?
Scrolling all the way to the top of the search history is no option for me.

I know this was requested on Zulip but because someone uses a custom mappin for the Trax encoder press, but the ~~default~~ most common mapping is [Library],GoToItem.

Reply:
The current system is also problematic for blind users as the tab order changes depending on the state of the search bar.

I'm ambivalent about the focus behaviour. What's the general stand towards this ?